Docker/README.md

```diff
@@ -0,0 +1,187 @@
+ports available etc.
+
+It has to be noted that the current docker-compose setup is mounting the [Docker/validators](validators)
+folder to the service containers in order to share the folders & files in it. That allow us to experiment/change
+configuration without having the need to rebuild the Docker image.
+
+### Docker-compose configuration
+
+The following environment variables are available to be used for each service on the
+[docker-compose.yml](docker-compose.yml) file configuration:
+* `NODE_TYPE` with values `primary|worker` . Defines the node type to bootstrap
+* `AUTHORITY_ID` with decimal numbers, for current setup available values `0..3`. Defines the
+id of the validator that the node/service corresponds to. Basically this defines which
+configuration to use under the `validators` folder.
+* `LOG_LEVEL` the level of logging for the node defined as number of `v` parameters (e.x `-vvv`). The following
+levels are defined according to the number of "v"s provided: `0 | 1 => "error", 2 => "warn", 3 => "info", 
+4 => "debug", 5 => "trace"`.
+* `CONSENSUS_DISABLED`, this value disables consensus (`Tusk`) for a primary node and enables the
+`gRPC` server. The value that should be passed is `--consensus-disabled`
+* `WORKER_ID` the id, as integer, for service when it runs as a worker
+* `CLEANUP_DISABLED` , when provided with value `true`, it will disable the clean up of the validator folder
+from the database & log data. This is useful to preserve the state between multiple docker compose runs.
+
+### Troubleshooting
+
+#### 1. Compile Errors when building Docker image
+If come across errors while the Docker image is being build, for example errors like:
+```
+error: could not compile `tonic`
+#9 373.3 
+#9 373.3 Caused by:
+#9 373.4   process didn't exit successfully: `rustc --crate-name tonic --edition=2018
+....
+#9 398.4 The following warnings were emitted during compilation:
+#9 398.4 
+#9 398.4 warning: c++: fatal error: Killed signal terminated program cc1plus
+#9 398.4 warning: compilation terminated.
+```
+
+it is possible that the Docker engine is running out of memory and there is no capacity to properly
+compile the code. In this case please try to increase the available RAM at least to 2GB and retry.
+
+#### 2. Mounts denied or cannot start service errors
+
+If you try to spin up the nodes via docker-compose and you come across errors such as `mounts dened`
+or `cannot start service`, please make sure that you allow Docker to share your host's [Docker/validators](validators)
+folder with the containers. If you are using Docker Desktop you can find more information of how to do
+that here: [mac](https://docs.docker.com/desktop/mac/#file-sharing), [linux](https://docs.docker.com/desktop/linux/#file-sharing),
+[windows](https://docs.docker.com/desktop/windows/#file-sharing)
+
+#### 3. Deprecated devicemapper storage driver
+
+If you see errors related to the `Deprecation of the devicemapper storage`, then you might need to
```

Comment:
FYI : those present as mount errors. I'd just be vague and point to Docker doc. This is fundamentally an upstream issue, I'm just mentioning it because the deprecation is recent and the forums are ablaze with it.